docs/source/config_file.rst

```diff
@@ -744,25 +752,28 @@ section of the command line docs.
 
     Make arguments prepended via ``Concatenate`` be truly positional-only.
 
+    .. warning::
+       Deprecated: use :confval:`extra_checks` instead.
+
 .. confval:: strict_equality
 
     :type: boolean
     :default: False
```

Comment:
I think the proper fix here might be to decrease the indentation here rather than increasing the indentation for the description -- RST markup uses 3-space indentation for these things typically

```suggestion
   :type: boolean
   :default: False
```

(same with `.. confval:: strict` below)